spec/features/events_spec.rb

```diff
@@ -166,6 +167,42 @@
     end
   end
 
+  scenario "logged in as Organizer I can send acceptance and then rejection emails and by that change the status notification flag" do
+    login(:organizer)
+    event = FactoryGirl.create(:event_with_accepted_applications, :in_selection_phase)
+
+    event.application_letters.each do |application|
+      application.status_notification_sent = false
+      application.save! if application.changed?
+    end
+  
+    applications = event.application_letters.select { | application_letter | application_letter.status == 'accepted' }
+    visit event_path(event)
+    click_button(I18n.t('events.applicants_overview.sending_acceptances'))
+    expect(page).to have_current_path(event_email_show_path(event_id: event.id), only_path: true)
+    fill_in :email_subject, with: "Subject Accepted"
+    fill_in :email_content, with: "Content Accepted"
+    click_button I18n.t('.emails.email_form.send')
+    expect(applications.size).to be > 0
```

Comment:
This line was mainly introduced because I used where instead of select before, which sometimes returned in 0 instead of the 5 expected applications, allthough they do exist. So I dont think its really needed, but I can move it up 1 line?